test/test_collection.py

```diff
@@ -118,10 +119,10 @@ def test_watch(self):
         res = self.explain.watch()
         self.assertIn("queryPlanner", res["stages"][0]["$cursor"])
         self.collection.watch(pipeline=[{"$project": {"tags": 1}}],
-                              batch_size=10, full_document="updateLookup")
+                               batch_size=10, full_document="updateLookup")
```

Comment:
This change seems off batch_size should align with pipeline.

Reply:
Done.